SONIC-DEV/sonic-buildimage
# [platform/cel-blackstone]: update drivers and plugins follow switchboard spec

**- What I did**

- Update Blackstone platform and device config follow switchboard spec

**- How I did it**
- Update I2C mapping 
- Implement COME CPLD driver
- Update sfputil follow MSIC CPLD spec

**- How to verify it**
- Check I2C Mapping with physical port
- Check QSFP reset/lpmode status while running the sfputil command.
